### ISSUE TITLE
i18n Time Issue

### ISSUE THREAD

**Issue by Jian-Min-Huang** [Jian-Min-Huang/tech-note#182]
* In MySQL 8, Database timezone settings only affect syntax like `SELECT now();`.
* We will append `serverTimezone=Asia/Taipei` in jdbc url. It affect the jdbc connection session time to database
* Backend service timezone setting will affect method like `LocalDateTime.now` and `OffsetDateTime.now(ZoneOffset.UTC)`
* service GMT+8, jdbc GMT+4, db GMT+0
* LocalDatetime and OffsetDateTime generate time means GMT+8, even OffsetDateTime shift to other timezone but db will save GMT+4 time
* When you browser data from db UI tool, the time means at GMT+4 (MySQL datetime doesn't store timezone info)
* When you read data from db, the service will calculate GMT+8 and GMT+4 and auto transform to GMT+8 time
* In conclusion, we need to set jdbc same as db for John's request
* p.s. Daniel has old way is we just save all time to Unix time for long
